README.md

```diff
@@ -1,70 +1,62 @@
-# Getting Started with Create React App
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+#  Space Travellers 
 
```

Comment:
[ OPTIONAL ] Please add instructions to run the React project locally. It should include commands like `npm install`, `npm start`, and `npm run test` to run tests.